File: web3rpc/rpc-specs/code-samples/javascript/kaia/filter/getFilterLogs.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const quantity = "0x16";
  provider.kaia
    .getFilterLogs(quantity, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/filter/getFilterLogs.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x16"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.get_filter_logs(quantity)

print(kaia_response)
